deploy/manifests/configmap.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: topolvm-config
+  namespace: topolvm-system
+data:
+  scheduler.yaml: |
+    default-divisor: 10
+    divisors:
+      myvg1: 10
```

I think this should be kept as the default (1) because this is intended for actual users.

```suggestion
    default-divisor: 1
```